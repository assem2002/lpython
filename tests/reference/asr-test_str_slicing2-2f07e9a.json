{
    "basename": "asr-test_str_slicing2-2f07e9a",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_str_slicing2.py",
    "infile_hash": "eb9027d2b6ec0aba9cc78aeedc1b0b36972ff36b076fecf547eeb014",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_str_slicing2-2f07e9a.stderr",
    "stderr_hash": "ad9eb0eea074a4462ebc45713d14d3b6e11e2c4ca86c82f053f53a25",
    "returncode": 2
}