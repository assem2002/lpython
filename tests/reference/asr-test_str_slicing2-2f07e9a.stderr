semantic error: slice indices must be integers or None
 --> tests/errors/test_str_slicing2.py:4:13
  |
4 |     print(s[1.5:3])
  |             ^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
